### PR TITLE
Revert "changed namespace from c to fabric"

### PR DESCRIPTION
**Pull request by mounderfod**
This fixes incompatibility as a result of the developer changing the tag namespace to `fabric:`, which no mod creates tags for. Please merge this, using `fabric:` does more harm than good.

### CODE REVIEW

**Comment by mounderfod** [comment]
Needs fixing, i will reopen it when fixed.

**Comment by mounderfod** [comment]
Fixed issues, ready to be merged again.

**Review by Shrimpstress — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -8,7 +8,7 @@ yarn_mappings=1.16.2+build.1
 loader_version=0.9.1+build.205
 
 # Mod Properties
-mod_version = 1.1
+mod_version = 1.0
```
Comment (UpcraftLP) [inline gradle.properties]: why revert the version?

File: src/main/resources/data/ferroustry/recipes/sapling2/lead_sapling.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   ],
   "key": {
     "#": {
-      "tag": "fabric:storage_blocks/lead"
+      "tag": "c:storage_blocks/lead"
```
Comment (UpcraftLP) [inline src/main/resources/data/ferroustry/recipes/sapling2/lead_sapling.json]: should be `c:lead_blocks`

File: src/main/resources/data/ferroustry/recipes/sapling2/tin_sapling.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   ],
   "key": {
     "#": {
-      "tag": "fabric:storage_blocks/tin"
+      "tag": "c:storage_blocks/tin"
```
Comment (UpcraftLP) [inline src/main/resources/data/ferroustry/recipes/sapling2/tin_sapling.json]: should be `c:tin_blocks`

**Review by UpcraftLP — CHANGES_REQUESTED** [review]

**Comment by mounderfod** [comment]
Fixed all those problems.

**Review by UpcraftLP — APPROVED** [review]

**Comment by mounderfod** [comment]
Just waiting on @Tfarcenim now, I highly doubt they'll approve the changes though.

**Review by UpcraftLP — APPROVED** [review]

**Comment by Tfarcenim** [comment]
I'll give an A for effort

**Pull request merged**